subprojects/docs/src/docs/userguide/dep-man/06-publishing/publishing.adoc

```diff
@@ -299,12 +295,44 @@ include::sample[dir="userguide/dependencyManagement/modelingFeatures/crossProjec
 include::sample[dir="userguide/dependencyManagement/modelingFeatures/crossProjectPublications/advanced-published/kotlin/buildSrc/src/main/kotlin/com/acme",files="InstrumentedJarsPlugin.kt[tags=add_variant_to_existing_component]"]
 ====
 
+[[sec:publishing_custom_artifacts_to_maven]]
+== Adding custom artifacts to a publication
+
+[WARNING]
```

Comment:
We don't need to address this now, but I think this text is really wrong now. If anything, it should be turned around starting with Gradle metadata and then mentioning limitations of poms.

> remember that they are _secondary_ artifacts that support a _primary_ artifact.

I think the destination between primary artifact and other artifact should no longer be made (or only in a note about pom specifics).

> Thinking about publications in this way should help you determine whether you should be adding custom artifacts to an existing publication, or defining a new publication.

This is also much to simple in in my mind a misleading advice. Adding additional variants or feature variants is the correct model in most cases.

One simple thing we can do is at least not mark this as a warning...

Reply:
I have reworked that WARNING block, let me know what you think!